src/platform-apps/channels/channel-view.tsx

```diff
@@ -21,14 +55,21 @@ export class ChannelView extends React.Component<Properties> {
     );
   }
 
+  renderMessages() {
+    const messagesByDay = this.getMessagesByDay();
+    const groupDays = Object.keys(messagesByDay).sort((a, b) => (a > b ? 1 : -1));
+
+    return <div className='container'>{groupDays.map((day: string) => this.renderDay(day, messagesByDay))}</div>;
```

Comment:
can we update this className to make it more specific? container has a lot of potential for conflicts as a class name. :)